src/specs/api.v2.yaml

```diff
@@ -2042,12 +2036,12 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/HTTPError"
-        "404":
-          description: The work results were not found.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/HTTPError"
+        # "404":
+        #   description: The work results were not found.
+        #   content:
+        #     application/json:
+        #       schema:
+        #         $ref: "#/components/schemas/HTTPError"
```

Comment:
if this code is not needed, can you delete it?

Reply:
done